### PR TITLE
Fix nodejs - server.close()

### PR DESCRIPTION
**Pull request by FlorianBELLAZOUZ**
Related to #241

### CODE REVIEW

File: nodejs/dist/uws.js [hunk 1]
```diff
@@ -500,6 +500,8 @@ class Server extends EventEmitter {
             this.httpServer.removeListener('upgrade', this._upgradeListener);
         }
 
+        if(this.httpServer) this.httpServer.close()
```
Comment (ronkorving) [inline nodejs/dist/uws.js]: Please try to protect code style of the project you contribute to. Compare this to the if-statement above.

**Comment by FlorianBELLAZOUZ** [comment]
#241

**Comment by alexhultman** [comment]
removeListener and close can both be under the same condition, no need to have two different branches. Otherwise kt shojld work just fine, thanks

**Comment by alexhultman** [comment]
Rip touch screen

File: nodejs/dist/uws.js [hunk 2]
```diff
@@ -498,6 +498,7 @@ class Server extends EventEmitter {
     close() {
         if (this._upgradeListener && this.httpServer) {
             this.httpServer.removeListener('upgrade', this._upgradeListener);
+            this.httpServer.close()
```
Comment (alexhultman) [inline nodejs/dist/uws.js]: Semicolon, please?

**Comment by alexhultman** [comment]
:+1:

**Pull request merged**